lib/src/sdk.ts

```diff
@@ -117,6 +136,8 @@ export class ExchangeSDK {
     customUrl?: string
   ) {
     this.providerId = providerId;
+    this.customUrl = customUrl;
```

Comment:
what's the point of this? just use the axiosClient defined with the base URL passed

Reply:
There are two different APIs for this functionality, i want to avoid getting them mixed up and being really explicit, this care will be removed hence this code will be removed as well in the future

Reply:
If this is a different url and you require multiple ones. Be more explicit on the name of this one and its usage. For me it looks like the url used in AxiosClient and there is no need to put in the state